Products/CMFDefault/skins/ursine/main_template.pt

```diff
@@ -3,15 +3,18 @@
   &quot;-//W3C//DTD XHTML 1.0 Transitional//EN&quot;
   &quot;http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd&quot;&gt;" />
 
-<html tal:define="globals context/@@ursine_globals"
+<html tal:define="globals context/@@ursine_globals;
+                  portal_state context/@@plone_portal_state;
+                  portal_url portal_state/portal_url"
```

Comment:
The `@@plone_portal_state` view is not defined in CMFDefault.  Instead, used `globals/portal_url`.